sandbox/src/mocks/r4/sendQuestionnaireResponse/requests/basicQuestionnaireResponse.json

```diff
@@ -0,0 +1,42 @@
+{
+  "resourceType": "QuestionnaireResponse",
+  "questionnaire": "https://fhir.nhs.uk/Questionnaire/ERS-ShortList",
+  "basedOn":  [
+    {
+      "reference": "ServiceRequest/a.832db7fa-ebdd-44b6-ab3b-8329c2d43149/_history/5"
```

Comment:
I see we have a question about including the version here already.

I suspect we may need this for optimistic locking. We can't escape that the shortlist is a sub-resource of the referral really.

Certainly the reference type does allow it http://hl7.org/fhir/R4/references-definitions.html#Reference.reference